src/main/java/org/nypl/findawayclientlibrary/PlaybackService.java

```diff
@@ -117,6 +114,7 @@ public void initPlaybackEngine() {
       playbackEngine = audioService.getAudioEngine().getPlaybackEngine();
     } catch (AudioEngineException e) {
       LogHelper.e(TAG, "Error getting playback engine: " + e.getMessage());
+      // TODO
```

Comment:
What's the TODO here?

Reply:
error handling that's more than printing to the log